pepper-apis/docs/specification/src/components/schemas.yml

```
@@ -1865,6 +1865,16 @@ Question.Matrix:
           enum:
             - MULTIPLE
             - SINGLE
+        renderModal:
+          type: boolean
+          default: false
+          description: When true, matrix question is wrapped inside a modal.
```

Shifting the description to describe what actions a client should take, instead of describing what the Angular client's behavior is.

```suggestion
          description: When true, the matrix question should be presented inside a modal box.
```